test/diagnostics/global-element.lua

```diff
@@ -0,0 +1,97 @@
+local config = require 'config'
+local util   = require 'utility'
+
+-- disable all default groups to make isolated tests
+config.set(nil, 'Lua.diagnostics.groupFileStatus', 
+{
+    ['ambiguity']     = 'None',
+    ['await']         = 'None',
+    ['codestyle']     = 'None',
+    ['strict-conventions']   = 'None',
```

Comment:
I don't think it is a good name.

Reply:
Do you have recommendations for a better name?

We would also like to contribute more diagnostics that might not match everyone's coding style going forward and add them to this group. But being honest, we struggled finding a good name and are open for suggestions!

Reply:
Maybe... just `conventions`.
This group feature will be deprecated in plan, so it is not important.

Reply:
Ok I'll change it to `conventions` then, thank you!